testutil/singlevm/setup.sh

```diff
@@ -327,3 +355,6 @@ fi
 #Also kill the CNCI (as there is no other way to delete it today)
 sudo killall qemu-system-x86_64
 sudo rm -rf /var/lib/ciao/instances
+sudo ip link del eth10
+sudo killall dnsmasq
```

Comment:
Regarding killing dnsmasq, I changed to killing dnsmasq by PID. I'm getting it from its pidfile